content/english/arduino/Circuit-Basics.md

```diff
@@ -1,35 +1,34 @@
-+ Breadboards allow us to easily add, remove, or reposition components. Below on the left is an image of a breadboard.
-+ The image to the right shows how the pins are connected to each other. 
-+ Each one of the numbered rows on the breadboard, shown in yellow, contains 5 pins holes. Every pin hole on a row (yellow line) is connected to the other pin holes on that line. 
-+ The red line on the right shows a common connection between those pin holes, while the blue line on the right side show a common connection between those pin holes. 
-+ The same is true on the red and blue lines on the left. However, the red and blue lines on the right and left are not connected to each other.
-+ Components that are plugged into the same common connection are connected in the circuit
-
-![Cannot load image](../img/img16.png)
-![Cannot load image](../img/img17.png)
-
-The Light Emitting Diode (or LED for short) is a small light that turns on when electricity flows through it. 
-
-+ The LED has 2 leads: the Anode and the Cathode.
-+ The Anode is longer than the Cathode and is always attached to the higher voltage.
-![Cannot load image](../img/img18.png)
-
-The resistor is a small piece of ceramic material that slows down the flow of electricity. 
-
-+ A resistor has 2 leads, but it is omnidirectional. In other words, it doesn’t matter which lead you attach to the higher voltage. 
-+ Most electronics can only handle a certain amount of electricity before they break, so resistors protect the other components in our game by keeping the electricity low. 
-![Cannot load image](../img/img19.png)
+---
+title: "Circuit Basics"
+description: ""
+draft: false
+hidden: false
+weight: 3
+---
+
+# Appendix
+
+## The components, in detail
+
+Imagine our circuit is a play. Before the final performance in front of an audience, we need a place to practice. This is where the breadboard comes in. 
++ Breadboards are simple platforms that we can use to build temporary circuits for testing and debugging before finalizing our design and permanently soldering (welding) it together. 
++ Breadboards allow us to easily add, remove, or reposition components. Below on the left is an image of a breadboard.
++ The image to the right shows how the pins are connected to each other. 
++ Each one of the numbered rows on the breadboard, shown in yellow, contains 5 pins holes. Every pin hole on a row (yellow line) is connected to the other pin holes on that line. 
++ The red line on the right shows a common connection between those pin holes, while the blue line on the right side show a common connection between those pin holes. 
++ The same is true on the red and blue lines on the left. However, the red and blue lines on the right and left are not connected to each other.
++ Components that are plugged into the same common connection are connected in the circuit
+
+![Cannot load image](../img/img17.png)
```

Comment:
Some notes about images:

- Can you change all the [Cannot load image] alt text into an appropriate description of the image?
- Renaming the images to match what they are talking about would be nice too
- Do the images look OK? Might have to use HTML and resize the images to an appropriate size.

Reply:
I think they look okay but I did rename the ones that are in use.